tests/integration/test_charm.py

```diff
@@ -22,6 +22,17 @@
 METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
 APP_NAME = "training-operator"
 CHARM_LOCATION = None
+APP_PREVIOUS_CHANNEL = "1.7/stable"
+
+PROMETHEUS_K8S = "prometheus-k8s"
```

Comment:
Can we switch to chisme abstraction here? WDYT? e.g. [example](https://github.com/canonical/kfp-operators/pull/505/files#diff-b76ee5f8f4cfcbcbb0c2e4224b3c7faf506e4f3d131f87fd9ebc2860996737b6)

Reply:
Definitely

Reply:
Updated, let me know what you think.